src/main/resources/biz_de.properties

```diff
@@ -266,7 +266,9 @@ EntityImportJobFactory.importMode.help = Gibt an, welche Zeilen der Datei berüc
 EntityImportSyncJobFactory.syncMode = Modus
 EntityImportSyncJobFactory.syncMode.help = Gibt an welche Datensätze berücksichtigt werden und was mit nicht-vorhandenen Datensätzen geschieht.
 EntityImportSyncJobFactory.syncSource = Quelle
-EntityImportSyncJobFactory.syncSource.help = Geben Sie hier die Datenquelle der importierten Daten an. Wird eine Datensynchronisation ausgewählt, so werden nur Daten aus der gleichen Quelle gelöscht und Daten aus anderen Quellen bleiben erhalten. Wenn Sie das Feld leer lassen und eine Synchronisation starten, werden Daten aus allen anderen Quellen gelöscht.
+EntityImportSyncJobFactory.syncSource.help = Geben Sie hier die Datenquelle der importierten Daten an.
```

Comment:
Wenn man nicht weiß was hier passiert, ist das ein bisschen knapp. Vorschlag:
Geben Sie hier die Datenquelle der importierten Daten an. Anhand der Quelle können Datensätze im Datensynchronisationmodus selektiv aktualisiert werden. Das bedeuted, ein Import mit einer Quelle erstellt Einträge mit dieser Quelle, und löscht auch nur Altdaten aus dieser Quelle.

Reply:
Es gab mehrere Runde mit mbr + aha rund um den Texte, aber ich mag dein Vorschlag.
@mbrsci was hältst du davon?

Reply:
Im Ticket haben wir gesagt der Text geht so: "Hier können Sie einen beliebigen Code eingeben um Ihre Datenquelle zu identifizieren, z.B. "ERP" oder "PIM". Die Quelle ist beim nächsten Import im Modus Synchronisieren relevant um zu bestimmen welche in Ihrer Importquelldatei nicht mehr vorhandenen Daten ggfs. bereinigt werden. Hinweis: Wenn Sie den Standardwert anpassen, empfehlen wir eine Job-Vorlage anzulegen um diese Quelle immer wiederzuverwenden!"